src/encrypt.js

```diff
@@ -1,10 +1,22 @@
 const Url = require('url-parse')
 const CryptoJS = require('crypto-js')
+const fs = require('fs')
 import { isUrl, isPatternUrl, isPatterObject, isPattern, isPath } from '../util'
 
+const configData = fs.readFileSync(__dirname + '/config.json', 'utf8')
```

Comment:
can this be template literal like below.

```
const configFile = 'config.json'
fs.readFileSync(`${__dirname}/${configFile}`, 'utf8')
```

Reply:
Yes, it can be. Refer [Comment](https://github.com/web-slate/crypto-urls/pull/50#discussion_r618096654)